src/main/java/com/example/coursework2/ExamController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.coursework2;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/exam/java")
+public class ExamController {
+    private final ExaminerServiceImpl examinerServiceImpl;
+
+    public ExamController(ExaminerServiceImpl examinerServiceImpl){
```

Comment:
```suggestion
    public ExamController(ExaminerService examinerServiceImpl){
```
инъекцию надо делать по интерфейсу